agent-ndk/src/main/java/com/newrelic/agent/android/ndk/ANRMonitor.kt

```diff
@@ -45,7 +45,11 @@ open class ANRMonitor {
                         return@Runnable
                     }
 
-                    runner.wait(ANR_TIMEOUT)
+                    try {
+                        Thread.sleep(ANR_TIMEOUT)
+                    } catch (e: Exception) {
+                        e.printStackTrace()
```

Comment:
This is good to know, but shoudl anything else be done in the monitor is a sleep interruption occurs?